rules/no-unused-expressions.js

```diff
@@ -35,6 +35,13 @@ function isInDoStatement(node) {
     return isInDoStatement(node.parent);
   }
 
+  if (
```

Comment:
Curious, can this be merged with the previous condition (L30)?

Reply:
It looks like we can completely remove condition at L30 at all, all tests are passing without it. L30 works for the subset of nodes, which L38 are also covering. I should check it before, sorry :)